src/researchhub_document/views/researchhub_unified_document_views.py

```diff
@@ -647,18 +590,37 @@ def test_get_unified_documents(self, request):
             "score",
             "fundraise",
         ]
+
         serializer = self.dynamic_serializer_class(
             page,
             _include_fields=_include_fields,
             many=True,
             context=context,
         )
+
         serializer_data = serializer.data
 
         return self.get_paginated_response(serializer_data)
 
     @action(detail=False, methods=["get"], permission_classes=[AllowAny])
     def get_unified_documents(self, request):
+
+        def get_page_url(page_number):
+            from urllib.parse import urlencode
+
+            from django.urls import reverse
```

Comment:
Nit: Not sure if it's worth scoping these imports (unnecessary clutter within actual code, failing imports are not recognized at module level).